packages/core/.eslintrc.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
     browser: true
   },
   rules: {
-    'multiline-comment-style': ['error', 'starred-block']
+    'multiline-comment-style': ['error', 'starred-block'],
```

Comment:
Needed?

Reply:
Maybe come back to `.eslintrc.js` later to make sure they're all the same/strip out unneeded globals